bindings/tflite/interpreter.c

```diff
@@ -427,6 +427,7 @@ TFL_CAPI_EXPORT extern void TfLiteInterpreterDelete(
   iree_vm_module_release(interpreter->hal_module);
   iree_vm_module_release(interpreter->user_module);
   iree_vm_instance_release(interpreter->instance);
+  iree_hal_driver_release((iree_hal_driver_t*)interpreter->driver);
```

Comment:
1. Release order should typically be the inverse of creation order. Creation order is `iree_vm_instance_create` -> `_TfLiteInterpreterPrepareHAL` -> `iree_vm_context_create_with_modules`. I think you should move this up one line.

2. No type cast should be needed here. `interpreter->driver` is already of type `iree_hal_driver_t*`

https://github.com/google/iree/blob/d03db58496a2dece4a80a28d35a93f59d0abd85d/bindings/tflite/interpreter.h#L21-L28

Reply:
Great, thanks for catching these things!